_episodes/wrap-up.md

```diff
@@ -129,6 +125,156 @@ Have a look at another example - a simplified directory structure of a blog webs
+Last time we did this in the settings, this time we will create a new branch
+called `gh-pages` which automatically turns on this functionality.
+
+Click on the button on the left-hand side of the page that shows a branching icon and
+says `main`.  
+Then in the box that reads "Find or create a branch" type `gh-pages`.
+Then click the "Create branch: gh-pages from 'main'" option that appears below.
+
+![GitHub branching dropdown menu with gh-pages typed in box](../fig/github-branch-ghpages.png){: height="300px"}
+
+Once you've clicked that option you will see the box change to say `gh-pages` instead
+of `main` indicating you are working on the new branch you created.
+Creating a `gh-pages` branch also has turned on the website to render at
+`<USERNAME>.github.io/mundana-theme-jekyll`.  You can navigate to `<USERNAME>.github.io/mundana-theme-jekyll`
+and check in "Settings" to see that it worked as expected.
+
+We will be making lots of edits to the new `gh-pages` branch in the challenges below.
+This will be easier to do if we make it the default branch.
+To change the `gh-pages` branch to the default branch, first click 
+the branches button to the right of the drop-down menu we used to create
+the `gh-pages` branch.
+Then in the box that shows the `Default branch` click the two arrows button
+on the upper right-hand side of the box.
+Click that two arrows button again on the next page, choose `gh-pages`
+from the drop down menu options, and click `Update`.
+Confirm your choice by clicking `I understand, update the default branch.`.
+Now when you return to the `Code` tab or the main page of your repo, you will see
+`gh-pages` in the branch drop down by default.
+
+The following series of challenges will have you practice the skills you
+learned in this lesson to modify this template.
+
+> ## Update GitHub About Section Link to Site
+> As we did before in the [Hosting Websites on GitHub episode](github-pages.md), update
+> the repository details to link directly to the rendering website
+> from your fork. 
+> 
+{: .challenge}
+
+> ## Customize the Configuration
+> Update the name and the description variables for the site.
+> Reload the rendered page to see the changes to the actual site.
+> 
+> Hint: See the [Starting with Jekyll episode](starting-jekyll.md) for review
+> 
+{: .challenge}
+
+> ## Customize the Header Menu
+> 
+> Find where the header menu is created and remove the "WP", "Ghost", and 
+> "Buy me a coffee" items.
+>
+> Hint: See the [Reusing Blocks of Content](includes.md) section for more information
+> about where to find the header menu.
+> 
+> > ## Solution
+> > 
+> > Remove lines 12-20 in the `_includes/menu-header.html` file.
+> > 
+> {: .solution}
+{: .challenge}
+
+> ## Update a Layout
+> 
+> NEED SUGGESTION FOR LAYOUT CHANGE 
+>
+> 
+{: .challenge}
+
+> ## Customize the About Page
+>
+> 1. Find the `About` page, update the layout to the `page` layout,
```

Comment:
Need some text clarification and then motivation couple sentences before the directions